public/docs/ts/_cache/tutorial/toh-pt5.jade

```diff
@@ -421,120 +503,171 @@ code-example(format='').
 
   ### Find our way back
 
   We can navigate *to* the `HeroDetailComponent` in several ways.
   How do we navigate somewhere else when we're done?
 
   The user could click one of the two links in the `AppComponent`. Or click the browser's back button.
-  We'll add a third option, a `goBack` method that navigates backward one step in the browser's history stack
+  We'll add a third option, a `goBack` method that navigates backward one step in the browser's history stack.
+
++makeExcerpt('app/hero-detail.component.ts', 'goBack')
 
-+makeExample('toh-5/ts/app/hero-detail.component.ts', 'go-back', 'app/hero-detail.component.ts (goBack)')(format=".")
+- var _CanDeactivateGuard = _docsFor == 'dart' ? '<em>routerCanDeactivate</em> hook' : '<em>CanDeactivate</em> guard'
+- var _CanDeactivateGuardUri = _docsFor == 'dart' ? 'angular2.router/CanDeactivate-class' : 'router/index/CanDeactivate-interface'
 .l-sub-section
- :marked
-   Going back too far could take us out of the application.
-   That's acceptable in a demo. We'd guard against it in a real application,
-   perhaps with the [*CanDeactivate* guard](../api/router/index/CanDeactivate-interface.html).
+  :marked
+    Going back too far could take us out of the application.
+    That's acceptable in a demo. We'd guard against it in a real application,
+    perhaps with the [!{_CanDeactivateGuard}](../api/!{_CanDeactivateGuardUri}.html).
+
 :marked
-  Then we wire this method with an event binding to a *Back* button that we add to the bottom of the component template.
-+makeExample('toh-5/ts/app/hero-detail.component.html', 'back-button')(format=".")
+  Then we wire this method with an event binding to a *Back* button that we
+  add to the bottom of the component template.
+
++makeExcerpt('app/hero-detail.component.html', 'back-button', '')
+
 :marked
-  Modifing the template to add this button spurs us to take one more incremental improvement and migrate the template to its own file
-  called `hero-detail.component.html`
-+makeExample('toh-5/ts/app/hero-detail.component.html', '', 'app/hero-detail.component.html')(format=".")
+  Modifing the template to add this button spurs us to take one more 
+  incremental improvement and migrate the template to its own file:
+
++makeExcerpt('app/hero-detail.component.html')
+
 :marked
   We update the component metadata with a `templateUrl` pointing to the template file that we just created.
-+makeExample('toh-5/ts/app/hero-detail.component.ts', 'template-url', 'app/hero-detail.component.ts (templateUrl)')(format=".")
+
++makeExcerpt('app/hero-detail.component.ts', 'templateUrl')
+
 :marked
-  Here's the (nearly) finished `HeroDetailComponent`:
-+makeExample('toh-5/ts/app/hero-detail.component.ts', 'v2', 'app/hero-detail.component.ts (latest)')(format=".")
+  Refresh the browser and see the results.
+  Here's the (nearly) finished `HeroDetailComponent` but without the `@Input` !{_decorator} on `hero`
```

Comment:
In fact, It's not removed very soon. Why not show it, with a note that we can take it out? If that doesn't work, just add a note after the sample.